### DISCUSSION TITLE
can not find some module in my exe

### DISCUSSION THREAD

**Discussion by David-dotcom666** [pyinstaller/pyinstaller#8694] [Help]
I used to frequently use pyinstaller to package my Python programs, but this time I encountered a problem. I used an open source project A and modified some code to install them into the site package using setup. py. However, after packaging, the package did not work properly and I did not find the folder with the package name in my folder. But I found that my other package B, which used pip install, was properly packaged and included in my project files. Additionally, I have tried adding A directly to the data, and exe works fine. But I don't want to use it in a way that exposes A's source code. Thank you for any suggestions and discussions.

**Comment by David-dotcom666** [comment]
BTW,i tried to use hiddenimports,but still show miss some module,seem like some module is ok,but some is not in the same package.and i add the missed module in the hiddenimports,it dose not work.

**Comment by bwoodsend** [comment]
This isn't much to go on but a good start would be to try installing it via `pip install .` rather than the deprecated `python setup.py install`.
